bears-app/src/components/shared/api/mock-data/content-data.json

```
@@ -0,0 +1,814 @@
+{
+  "lifeEventForm": {
+    "id": "death-of-a-loved-one",
+    "timeEstimate": "60 minutes",
+    "titlePrefix": "",
+    "title": "Benefit Finder After Losing A Loved One",
+    "summary": "We are sorry about your loss. You and your family may be eligible for survivor benefits from the government following a family member's death. Learn how to find the benefits you may qualify for.",
+    "lede": "",
+    "secondaryHeading": "Steps to find benefits",
+    "icon": "",
+    "sectionsEligibilityCriteria": [
+      {
+        "section": {
+          "id": "about-the-applicant",
+          "heading": "About the Applicant",
+          "description": "The applicant is the person who will receive the benefits.",
+          "fieldsets": [
+            {
+              "fieldset": {
+                "criteriaKey": "applicant_date_of_birth",
+                "legend": "Date of birth",
+                "required": "FALSE",
+                "hint": "For example: 4 28 1986",
+                "inputs": [
+                  {
+                    "inputCriteria": {
+                      "id": "applicant_date_of_birth",
+                      "type": "Date",
```

let's make this lowercase
```suggestion
                      "type": "date",
```